projects/ontimize-web-ngx/src/lib/components/expandable-container/o-expandable-container.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input, AfterViewInit } from '@angular/core';
+import { OServiceComponent } from '../o-service-component.class';
+
+export const DEFAULT_INPUT_O_EXPANDABLE_CONTAINER = [
```

Comment:
Consider using this inputs array in the  @Component definition and not declaring the two inputs using @Input (as we do in the other framework components)